Framework/Algorithms/src/CalculateTransmissionBeamSpreader.cpp

```diff
@@ -289,11 +289,11 @@ CalculateTransmissionBeamSpreader::fitToData(API::MatrixWorkspace_sptr WS) {
     b = std::pow(10, b);
     m = std::pow(10, m);
 
-    auto &X = result->x(0);
+    auto &X = result->points(0);
```

Comment:
I would remove the `&` to decrease the risk of confusion, since `points()` returns by value.